staging/src/k8s.io/kubectl/pkg/kuberc/kuberc_test.go

```diff
@@ -1475,6 +1441,9 @@ func TestApplyAliasBool(t *testing.T) {
 				if actualValue != expectedFlag.value {
 					t.Fatalf("unexpected flag value expected %t actual %s", expectedFlag.value, actualFlag.Value.String())
 				}
+				if originalCommand = actualCmd.Annotations[KubeRCOriginalCommandAnnotation]; !strings.Contains(originalCommand, expectedFlag.shorthand) {
```

Comment:
Same nit here, so it's easier to read:
```golang
originalCommand = actualCmd.Annotations[KubeRCOriginalCommandAnnotation]
if !strings.Contains(originalCommand, expectedFlag.shorthand) {
...
```
I reserve the "short-if form" for cases where you want the variable to exist only within the scope of that particular if-case.